docs-app/package.json

```diff
@@ -30,8 +30,8 @@
     "@babel/eslint-parser": "^7.19.1",
     "@crowdstrike/ember-toucan-styles": "^2.0.1",
     "@crowdstrike/tailwind-toucan-base": "^3.5.0",
-    "@docfy/core": "^0.7.0",
-    "@docfy/ember": "^0.7.0",
+    "@docfy/core": "^0.6.0",
+    "@docfy/ember": "^0.6.0",
```

Comment:
I had to bump these down as I was getting build errors (randomly) from docfy when running:

```bash
pnpm build
```

or

```bash
pnpm start
```

It's apparently not failing on CI though, which is weird.

I didn't see a helpful error as to _why_, so I figure reverting for the time being is the path of least resistance.  I'll pick this back up after other items come off of my plate.

Reply:
This was my PR that was part of the bump to version 0.7.0: https://github.com/josemarluedke/docfy/pull/161

But there were also a lot of 3 dependabot updates...https://github.com/josemarluedke/docfy/pulls?q=is%3Apr+is%3Aclosed